index.html

```diff
@@ -1,8 +1,118 @@
+      </article>
+      <article>
+        <p>Why Downtown Lancaster?</p>
+        <p>As Downtown Lancaster's arts, music, drinking &amp; dining scenes have garnered national recognition over the past decade, we firmly believe that the Lancaster tech scene is in the midst of its own big break out. By moving downtown beginning in 2016, we wanted CPOSC to be at the center of this impending tech explosion. As a bonus: it's an easy walk from the Ware Center to the CPOSC After Party!</p>
+      </article>
+      <article>
+        <p>What to expect:</p>
+        <p>Since its start, CPOSC has featured some of the most experienced technologists in the industry presenting on a variety of open source topics, from introductions to emerging technologies to deep dives into coding languages, as well as high-level case studies recounting what worked, what didn't work, and the lessons learned along the way.</p>
+      </article>
+    </section>
+    <section>
+      <h3>Conference Sponsors</h3>
+        <p>Thank you for supporting open source in Central Pennsylvania!</p>
+          <div>
+            Become a sponsor
+          </div>
+      <article class="sponsors">
+        <p>PLATINUM SPONSORS</p>
+          <img src="images/SeisanLogo.png">
+      </article>
+      <article class="sponsors">
+        <p>GOLD SPONSORS</p>
+          <img src="images/printedsolid.png">
+          <img src="images/simple.png">
+          <img src="images/linode.png">
+      </article>
+      <article class="sponsors">
+        <p>SILVER SPONSORS</p>
+          <img src="images/lulzbot.png">
+          <img src="images/TapestryLogo.png">
+          <img src="images/elastic-logo.png">
+      </article>
+      <article class="sponsors">
+        <p>BRONZE SPONSORS</p>
+          <img src="images/lullabot.png">
+      </article>
+      <article class="sponsors">
+        <p>HOST SPONSORS</p>
+          <img src="images/WareCenter.png">
+      </article>
+    </section>
+    <section>
+      <h4>confernce locations</h4>
+        <ul class="locationlist">
+          <li>The Ware Center</li>
+          <li>at Millersville University</li>
+          <li>42 N. Prince Street,</li>
+          <li>Lancaster, PA 17603</li>
+        </ul>
+        <ul class="locationlist">
+          <li>Prince St. Parking Garage</li>
+          <li>111 N. Prince Street,</li>
+          <li>Lancaster, PA 17603</li>
+        </ul>
+        <ul class="locationlist">
+          <li>Tellus360</li>
+          <li>24 E. King Street,</li>
+          <li>Lancaster, PA 17602</li>
+        </ul>
+      </section>
+        <div>
+        </div>
+    <footer>
+      <h5>Code of Conduct</h5>
+        <p>CPOSC is dedicated to a harassment-free conference experience for everyone. Our anti-harassment policy can be found on our About page, under Code of Conduct.</p>
+      <h6>Connect with CPOSC</h6>
+        <p>#CPOSC2017</p>
+    </footer>
   </body>
 </html>
```

Comment:
Everything looks great, the problem only occurs to extra editing and you need to implement your comments so I know why you made the decisions you chose.